vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,72 @@
+"""
+renders 3d point for 2d surfaces
+"""
+
+import math, copy
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+AXIS_X = "x"
+AXIS_Y = "y"
+AXIS_Z = "z"
+
+
+def convert_to_2d(x: int, y: int, z: int, scale: int, distance: int) -> tuple:
+    """
+    Converts 3d point to a 2d drawable point
+
+    >>> convert_to_2d(1, 2, 3, 10, 10)
+    (7, 15)
+
+    """
+    if type(x) is not int:
+        raise TypeError("x must be int")
+    if type(y) is not int:
+        raise TypeError("y must be int")
+    if type(z) is not int:
+        raise TypeError("z must be int")
+    if type(scale) is not int:
+        raise TypeError("scale must be int")
+    if type(distance) is not int:
+        raise TypeError("distance must be int")
+    projectedY = int(((y * distance) / (z + distance)) * scale)
+    projectedX = int(((x * distance) / (z + distance)) * scale)
+    return projectedX, projectedY
```

Comment:
```suggestion
    projected_x = int(((x * distance) / (z + distance)) * scale)
    projected_y = int(((y * distance) / (z + distance)) * scale)
    return projected_x, projected_y
```
As discussed in CONTRIBUTING.md, Python variable names should be in snake_case (all lowercase with underscores in between words).